sites/public/lib/HorizontalScrollSection.tsx

```diff
@@ -51,7 +51,9 @@ const HorizontalScrollSection = (props: HorizontalScrollSectionProps) => {
   return (
     <section className={props.className}>
       <div className={styles.title}>
-        {props.icon && <Icon size="xlarge" symbol={props.icon} className={styles.icon} />}
+        {props.icon && (
```

Comment:
Are the parens here necessary? They weren't there before.

Reply:
Yeah I believe that's the correct style. Not sure why they weren't there before